src/pick.js

```diff
@@ -9,7 +8,7 @@ var _indexOf = require('./internal/_indexOf');
  * @func
  * @memberOf R
  * @category Object
- * @sig [String] -> {String: *} -> {String: *}
+ * @sig [k] -> {k: v} -> {k: v}
```

Comment:
We document *intended* use in the type signatures. We often use `[a]`, for example, though the elements of an array needn't be of the same type.